File: src/main/java/com/ptah/camunda/delegate/SayGoodbyeDelegate.java
package com.ptah.camunda.delegate;

import org.camunda.bpm.engine.delegate.DelegateExecution;
import org.camunda.bpm.engine.delegate.JavaDelegate;
import org.springframework.stereotype.Component;

@Component("SayGoodbyeDelegate")
public class SayGoodbyeDelegate implements JavaDelegate {
    @Override
    public void execute(DelegateExecution execution) throws Exception {
        execution.setVariable("Hello","bye bye");
    }
}

File: backend/src/main/kotlin/com/github/jarryzhou/ptah/camunda/delegate/SayGoodbyeDelegate.kt
package com.github.jarryzhou.ptah.camunda.delegate

import org.camunda.bpm.engine.delegate.DelegateExecution
import org.camunda.bpm.engine.delegate.JavaDelegate
import org.springframework.stereotype.Component

@Component("SayGoodbyeDelegate")
class SayGoodbyeDelegate : JavaDelegate {
    @Throws(Exception::class)
    override fun execute(execution: DelegateExecution) {
        execution.setVariable("Hello", "bye bye")
    }
}
